File: week11/week11-1.py
# week11-1.py Hash Map , Hash Set
table = {} #用大括號但代表字典 ，也就是 Hash Table (Hash Map , Hash Set)
table[9977341] = 9
table[4433997] = 4

print(table[9977341]) #會印出9
print(table[4433997]) #會印出4
#print(table[0]) #沒有這個key 會爆炸
if 0 not in table:
  print('table[0]不存在')

File: week11/week11-1.cpp
#include <iostream>
#include <unordered_map>
///因為是++2021 不能直接在codeblock使用
///setting-compiler,加入c++11就可以了
using namespace std;

int main()
{
        unordered_map<int,int> table;
        table[9977341] = 9;
        table[4433997] = 4;
        cout << table[9977341] << endl;
        cout << table[4433997] << endl;
}
